### DISCUSSION TITLE
How to Display Events for Multiple Organizations on a single Pretix Instance?

### DISCUSSION THREAD

**Discussion by ccccc382** [pretix/pretix#4013] [Q&A]
We are currently using Pretix to host events for several student culture groups, each represented as a separate organization within our Pretix instance. Our aim is to create a simple overview that showcases events from all these organizations in one place.

Issue:
We've encountered limitations when attempting to achieve this through both the Pretix API and the embeddable widget:

API: The API seems to require separate authentication per organization, hindering our ability to retrieve events across multiple organizations seamlessly.
Embeddable Widget: While the embeddable widget works well for showcasing events for a single organization, we haven't found a way to configure it to display events from multiple organizations simultaneously.

One possible solution would be to write code that makes multiple api calls to each individual api key. 
but we are wondering if there is a simpler solution.

**Comment by raphaelm** [comment]
There is no such feature currently

**Reply by akolaarthurali** [reply]
I think if the time comes and it is implemented, it will be a great feature and addition to pretix. Pretix will have a social media kind of feel. (Where all posts are in a single user interface just like it is with social media sites like x(twitter) or mastodon- which has some resemblance to pretix for they are all self-hosted. 

This feature presents a future where pretix transitions into a social media platform for all events created with pretix
